CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file. See [standa
 
 ## [0.11.0](https://github.com/getditto/react-ditto/compare/v0.11.0-alpha.0...v0.11.0) (2023-03-22)
 
+### Unreleased
+
+## Changed
+
+- browser support now matches the `@dittolive/ditto` package. see [Ditto's
+  documentation](https://docs.ditto.live/compatibility/js-web)
+  ([#56](https://github.com/getditto/react-ditto/pull/56))
+
```

Comment:
Sorry for the lack of docs in this regard, but for the last few releases, we've been using conventional commits and [`standard-version`](https://github.com/conventional-changelog/standard-version) to generate this changelog. I tried running it again (`npx standard-version`) to see what would happen with this manually added entry and it resulted in this:

```md
### [0.11.1](https://github.com/getditto/react-ditto/compare/v0.11.0...v0.11.1) (2024-04-18)


### Bug Fixes

* remove implicit dependency on `lodash` ([9b56f93](https://github.com/getditto/react-ditto/commit/9b56f9331de4e5718478717d0c9527dca2764d15))

## [0.11.0](https://github.com/getditto/react-ditto/compare/v0.11.0-alpha.0...v0.11.0) (2023-03-22)

### Unreleased

## Changed

- browser support now matches the `@dittolive/ditto` package. see [Ditto's
  documentation](https://docs.ditto.live/compatibility/js-web)
  ([#56](https://github.com/getditto/react-ditto/pull/56))
```

Since this PR mainly focuses on upgrading dependencies, I wouldn't want to rewrite the whole PR's commit message when merging it just to see a new entry in the changelog. Is there a chance we could separate this change to use a properly formatted commit message? Let me know if you have other ideas.

Reply:
Sure, I just saw the changelog and wanted to add to it. I will remove that change and update the PR title to use the standard-version syntax.

Reply:
Hmm, adding the markdown link to the documentation in the PR title sure looks wonky. I can squash-merge the PR with a commit message

`chore: match supported browsers with the `@dittolive/ditto` package. see [Ditto's documentation](https://docs.ditto.live/compatibility/js-web) for details.`

does that work?

Reply:
Yeah, I think that works. I'd just like to point out that only commits marked with `feat` and `fix` will make it to the changelog, so if you want to have that message show there you should pick one of those; if not, `chore` seems fine to me too.